settings.py

```diff
@@ -504,9 +504,8 @@ def JINJA_CONFIG():
     from django.core.cache.backends.memcached import MemcachedCache
     from django.core.cache import caches
     cache = caches['memcache']
-    config = {'extensions': ['jinja2.ext.i18n', 'tower.template.i18n',
-                             'jinja2.ext.with_', 'jinja2.ext.loopcontrols',
-                             'jinja2.ext.autoescape',
+    config = {'extensions': ['jinja2.ext.i18n', 'jinja2.ext.with_',
```

Comment:
You shouldn't need both. The puente one subclasses the other one. So this
situation is very fishy....
On Oct 30, 2015 7:48 PM, "Rob Hudson" <notifications@github.com> wrote:

> In settings.py
> <https://github.com/mozilla/kuma/pull/3613#discussion_r43563466>:
>
> > @@ -504,9 +504,8 @@ def JINJA_CONFIG():
> >      from django.core.cache.backends.memcached import MemcachedCache
> >      from django.core.cache import caches
> >      cache = caches['memcache']
> > -    config = {'extensions': ['jinja2.ext.i18n', 'tower.template.i18n',
> > -                             'jinja2.ext.with_', 'jinja2.ext.loopcontrols',
> > -                             'jinja2.ext.autoescape',
> > +    config = {'extensions': ['jinja2.ext.i18n', 'jinja2.ext.with_',
>
> Somehow that broke everything b/c gettext isn't in the context? Also, the
> example jingo test project has both - do you need both? I tried it and it
> failed. I'll poke with this on Monday.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/kuma/pull/3613/files#r43563466>.
>